src/cli/pad.rs

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Matteo Dell'Acqua
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+ */
+
+use crate::prelude::*;
+use anyhow::{ensure, Context, Result};
+use clap::{ArgMatches, Args, Command, FromArgMatches, ValueEnum};
+use common_traits::UnsignedInt;
+use log::info;
+use std::{
+    ffi::OsString,
+    mem::size_of,
+    path::{Path, PathBuf},
+};
+
+pub const COMMAND_NAME: &str = "pad";
+
+fn pad(path: impl AsRef<Path>, block_size: u64) -> Result<()> {
```

Comment:
There's no reason for this to be a `u64`, `usize`is the standard integer type unless you have specific size problems.